app/src/main/res/layout/activity_profile.xml

```diff
@@ -0,0 +1,169 @@
+        app:layout_constraintTop_toTopOf="parent"
+        tools:text="0" />
+
+    <ImageButton
+        android:id="@+id/ibCommentBubble"
+        android:layout_width="44dp"
+        android:layout_height="42dp"
+
+        android:layout_marginStart="4dp"
+        android:layout_marginTop="108dp"
+
+
+        android:adjustViewBounds="true"
+        android:background="@drawable/comment_bubble"
+        android:contentDescription="@string/ibCommentBubble"
+        android:scaleType="centerCrop"
+        android:scaleX="0.85"
+        android:scaleY="0.85"
+        android:src="@drawable/comment_bubble"
+        app:layout_constraintStart_toEndOf="@+id/tvCommentCounter"
+        app:layout_constraintTop_toTopOf="parent"
+        app:srcCompat="@android:color/holo_blue_dark"
+        tools:srcCompat="@drawable/comment_bubble" />
+
+    <TextView
+        android:id="@+id/tvUserName"
+        android:layout_width="200dp"
+        android:layout_height="58dp"
+        android:layout_marginStart="8dp"
+        android:layout_marginTop="16dp"
+        android:text="@string/tvUserName"
+        android:textColor="@android:color/black"
+        android:textSize="24sp"
+        app:layout_constraintStart_toEndOf="@+id/ivProfilePhoto"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <TextView
+        android:id="@+id/tvGenderAge"
+        android:layout_width="143dp"
+        android:layout_height="28dp"
+        android:layout_marginStart="8dp"
+        android:layout_marginTop="4dp"
+        android:layout_marginEnd="8dp"
+        android:text="@string/tvGenderAge"
+        android:textColor="@android:color/black"
+        android:textSize="18sp"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHorizontal_bias="0.0"
+        app:layout_constraintStart_toEndOf="@+id/ivProfilePhoto"
+        app:layout_constraintTop_toBottomOf="@+id/tvUserName" />
+
+    <TextView
+        android:id="@+id/tvAlias"
+        android:layout_width="70dp"
+        android:layout_height="27dp"
+        android:layout_marginStart="25dp"
+        android:layout_marginTop="8dp"
+        android:contentDescription="@string/tvAlias"
+        android:text="@string/tvAlias"
+        android:textAlignment="center"
+        android:textColor="@android:color/black"
+        android:textSize="24sp"
+        android:textStyle="bold"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/ivProfilePhoto"
+        tools:text="@string/tvAlias" />
+
+    <TextView
+        android:id="@+id/tvUserAlias"
+        android:layout_width="258dp"
+        android:layout_height="27dp"
+        android:layout_marginTop="8dp"
+        android:text="@string/tvUserAlias"
+        android:textAlignment="center"
+        android:textSize="24sp"
+        app:layout_constraintStart_toEndOf="@+id/tvAlias"
+        app:layout_constraintTop_toBottomOf="@+id/ivProfilePhoto" />
+
+    <Button
```

Comment:
This should be hidden for now, other three shown buttons are shown on wireframe.
Also, consider putting button text in lowercase.